### PR TITLE
made schema

### PR DESCRIPTION
**Pull request by nicktran6997**
Currently some ambiguity in code (since haven't integrated from frontend to backend) so some pieces a bit out of place.

### CODE REVIEW

**Review by benihime1458 — APPROVED** [review]
Hey Nick. Nice starter schema setup. I'll base the mock UI data accordingly.

File: server/models/cards.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const mongoose = require('mongoose')
+const cardSchema = new mongoose.Schema({
+    name: {type: String, required: true}, 
+    desc: {type: String, required: true},
+    column: {type: mongoose.Schema.Types.ObjectId, ref: "Column"},
+    deadline: Date,
```
Comment (swathivottigunta) [inline server/models/cards.js]: Notes/Comment and also tag property type - missing

Comment (nicktran6997) [reply server/models/cards.js]: The ticket says that I don't have to implement those yet. I also removed the deadline portion as well. I guess we'll be adding those other pieces later.

Comment (sina-jamshidi) [reply server/models/cards.js]: We will be adding those later! This level of detail is fine for now

**Comment by nicktran6997** [comment]
One feature currently left out concerning users + board creation

**Comment by nicktran6997** [comment]
The GET routes were not required by the ticket, but I made them hastily for debugging with postman. AND WHOOPS MB ABOUT THE MONGOURI.

**Pull request merged**